README.md

```diff
@@ -7,6 +7,22 @@
 
 The Go [OpenTelemetry](https://opentelemetry.io/) implementation.
 
+## Project Status
+
+**Warning**: this project is currently in a pre-GA phase. Backwards
+incompatible changes may be introduced in subsequent minor version releases as
+we work to track the evolving OpenTelemetry specification and user feedback.
+
+Our progress towards a GA release candidate is tracked in [this project
+board](https://github.com/orgs/open-telemetry/projects/5). This release
+candidate will follow semantic versioning and will be released with a major
+version greater than zero.
```

Comment:
Will the RC be released with a non-zero major version or only the ultimate GA release?  I think [semver would allow 1.0.0-rc1](https://semver.org/#spec-item-11) but I'm not sure how Go modules would handle that.

Reply:
Pre-releases are understood by Go modules, and https://blog.golang.org/publishing-go-modules#TOC_3. says:

> You can specify pre-release versions by appending a hyphen and dot separated identifiers (for example, v1.0.1-alpha or v2.2.2-beta.2). Normal releases are preferred by the go command over pre-release versions, so users must ask for pre-release versions explicitly (for example, go get example.com/hello@v1.0.1-alpha) if your module has any normal releases.

This means that when `v1.0.0-rc1` is tagged, `go get go.opentelemetry.io/otel` will _not_ select that version, but the `v0.13.0` tag instead (or whichever the latest `v0.x` tag there is at the time). I think this behaviour is the least-surprising to most developers.